src/ceph-disk/ceph_disk/main.py

```diff
@@ -1010,30 +1025,44 @@ def allocate_osd_id(
     cluster,
     fsid,
     keyring,
+    path,
 ):
-    """
-    Accocates an OSD id on the given cluster.
-
-    :raises: Error if the call to allocate the OSD id fails.
-    :return: The allocated OSD id.
```

Comment:
this docstring seems useful and still true? if the workflow has really changed where this doesn't match, maybe update it to reflect that?